apps/perf-test/tasks/flamegraph/helpers.js

```diff
@@ -0,0 +1,17 @@
+// TODO: test regex on mac, linux and windows
+const frameworkName = new RegExp(
+  /~([a-zA-Z]{1}|[a-zA-Z]{2}|[a-zA-Z]{1}|[a-zA-Z]{1}\.[a-zA-Z]{1})[\. ]|~render |~\(anonymous\)|\(unknown\)|\~webpack/
+);
+const systemName = new RegExp(/\(C\+\+\)|\(lib\)/);
+
+function isFrameworkName(name) {
+  return name.match(frameworkName);
```

Comment:
If you just need the boolean, not the match object, you can use `RegExp.prototype.test` (should be marginally more efficient).
```suggestion
  return frameworkName.test(name);
```